lib/compat/wordpress-6.0/class-gutenberg-rest-pattern-directory-controller.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Global_Styles_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Controller which provides REST endpoint for block patterns.
+ */
+class Gutenberg_REST_Pattern_Directory_Controller extends WP_REST_Pattern_Directory_Controller {
+	/**
+	 * Search and retrieve block patterns metadata
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 *
+	 * @return WP_Error|WP_REST_Response Response object on success, or WP_Error object on failure.
+	 */
+	public function get_items( $request ) {
+		/*
+		* Include an unmodified `$wp_version`, so the API can craft a response that's tailored to
+		* it. Some plugins modify the version in a misguided attempt to improve security by
+		* obscuring the version, which can cause invalid requests.
+		*/
```

Comment:
Indentation:

```suggestion
		/*
		 * Include an unmodified `$wp_version`, so the API can craft a response that's tailored to
		 * it. Some plugins modify the version in a misguided attempt to improve security by
		 * obscuring the version, which can cause invalid requests.
		 */
```

Reply:
Good catch! It seems this needed fixing for almost all comments in this file.